packages/cli/bin/cli.js

```diff
@@ -88,6 +88,7 @@ const argv = yargs
   .recommendCommands()
   .demandCommand(1, '')
   .completion()
+  .wrap(yargs.terminalWidth() * 0.8)
```

Comment:
@drewjenkins Instead of always making it 80%, I wonder if we should approach this sort of like response web design where we have breakpoints so that when a terminal is narrow we use the full-width and only switch to a percentage (or larger fixed width) at larger widths.

Reply:
Good call, fixed